generative/2d_ldm/config/config_train_32g.json

```diff
@@ -0,0 +1,61 @@
+{
+	"channel": 0,
```

Comment:
Please align the `channel` with other lines.

Thanks.

Reply:
Done. Thanks